docs/stories/components/charts/LLinearChartExpand.stories.js

```diff
@@ -10,7 +10,9 @@ export default {
     loading: { control: 'boolean', description: 'Expanded chart loading: true / not loading: false' },
     expand: { action: 'expand', description: 'Toggle chart expanded when "Recolher" clicked' },
     search: { action: 'search', table: {disable: true} },
-    sectionAfterValue: { description: 'Slot can show extra content and return info to parent components to show more details' }
+    eventClick: { action: 'eventClick', description: 'Event emitted activated when label is clicked' },
```

Comment:
Sugestão de alteração para description: 'Event emitted when label is clicked" (sem o activated)